docs/source/llms/tracing/index.rst

```diff
@@ -408,6 +413,32 @@ for model/API invocations to the active MLflow Experiment.
             :width: 100%
             :align: center
 
+    .. tab:: Gemini
```

Comment:
Can we add a small note that async APIs like `generate_content_async` are not supported yet?

Reply:
Good call. Added the disclaimer to the documentation.